api/controllers/Irc_EntryController.js

```diff
@@ -55,6 +57,12 @@ let processEventDetainee = (request_body) =>
   Centres.findOne({ name: request_body.centre })
     .then((centre) => createOrUpdateDetainee(generateDetainee(centre, request_body)));
 
+let publishCentreUpdates = (centre) =>
+  Centres.findOne(centre)
+    .then(BedCountService.performConfiguredReconciliation)
+    .then((centre) => Centres.publishUpdate(centre.id, centre.toJSON()))
```

Comment:
If you `tap` this line (and possibly the one above) instead, you shouldn't need the one below... or you could use `return` to explicitly specify the resolution of the promise chain.